lib/netext/httpext/response_type_gen.go

```diff
@@ -60,7 +60,7 @@ func (i ResponseType) MarshalJSON() ([]byte, error) {
 func (i *ResponseType) UnmarshalJSON(data []byte) error {
 	var s string
 	if err := json.Unmarshal(data, &s); err != nil {
-		return fmt.Errorf("ResponseType should be a string, got %s", data)
```

Comment:
This is generated so you shouldn't edit it manually

Reply:
Thanks, I reverted the change.